ui/src/components/settings/BasicSettings.vue

```diff
@@ -223,7 +223,7 @@
                 } else {
                     localStorage.removeItem("defaultNamespace")
                 }
-                this.$toast().saved();
```

Comment:
I'm not sure it's worth creating a new type of toast.
Maybe an alternative would be that you use the options from the saved method to give an "genericQualifier" key which defines whether or not we should display the saved thing in italic (false = italic, which will be the default not to break anything).

You can then update the saved() method to check for this param (don't forget to be null-safe on the options param :+1: ). Conditionally on that you can create a new translation that **doesn't includes** the `<em>` (see [here](https://github.com/kestra-io/kestra/blob/develop/ui/src/translations.json#L27)) if this param is true.

This way we now have a customizable toast which allow to display saved things whether they are id or not (and so the display will be adapted to it).

An example of usage for settings saving would then be
`this.$toast().saved(this.$t('setting'), undefined, {genericQualifier: true})`

To recap:

- in saved method check for options?.genericQualifier
- if true -> use $t('generic saved done') else -> use $t('saved done')
- create a translation for `generic saved done` key [here](https://github.com/kestra-io/kestra/blob/develop/ui/src/translations.json)
- use `this.$toast().saved(this.$t('setting'), undefined, {genericQualifier: true})` everywhere a setting is saved
- remove your introduced "toast().settingsSaved()"

Reply:
Ok.I will update it.